gammapy/maps/axes.py

```diff
@@ -1734,7 +1746,7 @@ def drop(self, axis_name):
         Returns
         -------
         axes : `MapAxes`
-            Axes with squashed axis.
+            Axes without the axis with name `axis_name`.
```

Comment:
Not sure of the meaning here?

Reply:
It means that it returns a `MapAxes` without the axis that the user asked to `drop` by using the name of this axis to remove it. 
maybe `Axes without the axis nameed axis_name.` 
Or if you have any other wording proposition !

Reply:
The wording is not completely understandable. Maybe: `Axes without the axis_name?`